### PR TITLE
Adds quick commit author overriding

### PR DESCRIPTION
**Pull request by exactly-one-kas**

### CODE REVIEW

File: src/ui/DetailView.cpp [hunk 1]
```diff
@@ -1297,4 +1307,72 @@ void DetailView::findPrevious()
...
+  if (!mOverrideUser.isEmpty() || !mOverrideEmail.isEmpty())
+    text += " (<a href=\"reset\">" + tr("reset") + "</a>)";
+
+  mAuthorLabel->setText(text);
+}
+
+void DetailView::authorLinkActivated(const QString &href)
+{
+  if (href == "changeAuthor") {
+    QDialog *dialog = new QDialog(this);
```
Comment (Murmele) [inline src/ui/DetailView.cpp:1345]: Adding a title?
Or a description what happens when the user changes something in this dialog?

File: src/ui/DetailView.cpp [hunk 2]
```diff
@@ -1297,4 +1307,72 @@ void DetailView::findPrevious()
...
+    QDialogButtonBox *buttons = new QDialogButtonBox(dialog);
+    buttons->addButton(QDialogButtonBox::Ok);
+    buttons->addButton(QDialogButtonBox::Cancel);
+    connect(buttons, &QDialogButtonBox::accepted, dialog, &QDialog::accept);
+    connect(buttons, &QDialogButtonBox::rejected, dialog, &QDialog::reject);
+    layout->addRow(buttons);
+
+    connect(dialog, &QDialog::accepted, [this, userEdit, emailEdit]() {
+      mOverrideUser = userEdit->text();
+      mOverrideEmail = emailEdit->text();
```
Comment (Murmele) [inline src/ui/DetailView.cpp:1363]: Should we store the data in the config, so it is stored for that project and will not be reset every time?

**Review by Murmele — CHANGES_REQUESTED** [review]

File: src/ui/DetailView.cpp (hunk 2 above)
Comment (exactly-one-kas) [reply src/ui/DetailView.cpp:1363]: Email and author are already saved per repository. This only adds ad-hoc overriding

Comment (Murmele) [reply src/ui/DetailView.cpp:1368]: ok

File: src/ui/DetailView.cpp (hunk 1 above)
Comment (Murmele) [reply src/ui/DetailView.cpp:1345]: @exactly-one-kas Please write done or something that I know you changed this. Thanks :)

**Pull request merged**